src/Client.cpp

```diff
@@ -150,6 +150,13 @@ err_code_t Client::version(types::broadcast_result_t** results, size_t* nHosts)
 }
 
 
+err_code_t Client::quit() {
+  broadcastCommand(keywords::kQUIT, 4);
```

Comment:
@lembacon @zzl0  This changeset is excluded now since it's OT, not so emergent, and there're more than one solution. Would please send the patch in another PR ?